roles/cifmw_cephadm/tasks/monitoring.yml

```diff
@@ -14,40 +14,44 @@
 # License for the specific language governing permissions and limitations
 # under the License.
 
-- name: Create dashboard spec
-  ansible.builtin.template:
-    src: templates/ceph_monitoring_stack.yml.j2
-    dest: "{{ cifmw_ceph_dashboard_spec_path }}"
-    mode: '0644'
-    force: true
-
-- name: Config ssl cert(s) and key(s) for the exposed components
-  become: true
-  when: cifmw_cephadm_dashboard_crt | length > 0 and
-        cifmw_cephadm_dashboard_key | length > 0
+- name: Build and apply the monitoring stack daemons
+  when:
+    - cifmw_cephadm_dashboard_enabled | bool
```

Comment:
I'm wondering if, instead of changing the code in the role itself, it's better to have a variable in the ceph playbook [1] and skip the dashboard entirely.

[1] https://github.com/openstack-k8s-operators/ci-framework/blob/main/playbooks/ceph.yml#L276

Reply:
:+1: would be better indeed

Reply:
yes we can do that but we should also update the ceph_overrides in the documeantion: https://ci-framework.pages.redhat.com/docs/ci-framework/05_deploy_va.html#ceph